projects/plugins/jetpack/_inc/client/newsletter/index.jsx

```diff
@@ -50,11 +45,13 @@ function Subscriptions( props ) {
 			{ foundSubscriptions && (
 				<>
 					<SubscriptionsSettings siteRawUrl={ siteRawUrl } blogID={ blogID } />
-					{ isEmailSettingsEnabled && <EmailSettings /> }
-					{ isNewsletterCategoriesEnabled && <NewsletterCategories /> }
+					<EmailSettings />
+					<NewsletterCategories />
 					<MessagesSetting { ...props } />
 				</>
 			) }
+			<NewsletterCategories />
```

Comment:
This is causing the NewsletterCategories component to be rendered twice.

Reply:
Actually Messages too.

Reply:
That's a good point! In the latest rebase, those were moved under the `foundSubscriptions` condition. I'll change it.